snapshots/countries/ID_COMMON.json

```diff
@@ -1197,7 +1197,7 @@
     "2029-12-03": "Isra' and Mi'raj (estimated)",
     "2029-12-25": "Christmas Day",
     "2030-01-01": "New Year's Day",
-    "2030-02-03": "Lunar New Year (estimated)",
+    "2030-02-02": "Lunar New Year (estimated)",
```

Comment:
I can't find the official source for 2030 yet - although the last time this happens in 2007 they were observed on Jan, 18 instead as per `CHINESE_CALENDAR` 👀

https://news.detik.com/berita/d-641866/inilah-hari-libur-cuti-bersama-2007

Reply:
Perhaps they don't do their own calculations, but rely on the China calendar. Let's wait until 2030.

Reply:
The fact that I've seen Indonesia-based pages on FB talking about [Kongsi Raya](https://en.m.wikipedia.org/wiki/Kongsi_Raya)/"Hari Raya Idul Imlek" long holidays like their BN/MY/SG compatriots probably didn't help with such confusion as well